cmd/zoekt-sourcegraph-indexserver/index.go

```diff
@@ -272,6 +275,13 @@ func gitIndex(c gitIndexConfig, o *indexArgs) error {
 
 	debug.Printf("successfully fetched git data for %q (%d commit(s)) in %s", o.Name, successfullyFetchedCommitsCount, fetchDuration)
 
+	logger.Info("successfully fetched git data",
```

Comment:
@ggilmore Just to confirm, is Info level the preference for both of our new zap log messages?